migrations/Version20220204135456.php

```diff
@@ -0,0 +1,91 @@
+<?php
```

Comment:
No, You don't update any entity, you shouldn't send me a new Migration. Don't send this file

Reply:
I removed the migration, thank you